### ISSUE TITLE
Manifest Merging Errors on Attribute activity#com.facebook.FacebookActivity@theme

### ISSUE THREAD

**Issue by kirtan403** [firebase/FirebaseUI-Android#400]
* Android device: Xiomi Redmi 3s Prime
  * Android OS version: 6.0.1
  * Google Play Services version: 9.8.77
  * Firebase/Play Services SDK version: 9.8.0/3.0.0
  * FirebaseUI version: 1.0
 

When I go to merged Manifest file, I see the following error:

> Merging Errors: Error: Attribute activity#com.facebook.FacebookActivity@theme value=(@android:style/Theme.Translucent.NoTitleBar) from AndroidManifest.xml:32:13-72 is also present at AndroidManifest.xml:78:13-58 value=(@style/FirebaseUI.Translucent). Suggestion: add 'tools:replace="android:theme"' to <activity> element at AndroidManifest.xml:29:9-32:75 to override. app main manifest (this file), line 31

Now, I didn't used any facebook library, I think its coming from firebase ui for auth.

I also tried editing <application> tag by adding `tools:node="merge"` and `tools:replace="android:theme"` but it, still gives the same error.

My gradle dependencies:
```
    compile fileTree(dir: 'libs', include: ['*.jar'])
    testCompile 'junit:junit:4.12'
    compile 'com.android.support:appcompat-v7:25.0.0'
    compile 'com.android.support:design:25.0.0'
    compile 'com.google.firebase:firebase-core:9.8.0'
    compile 'com.google.firebase:firebase-auth:9.8.0'
    compile 'com.google.firebase:firebase-crash:9.8.0'
    compile 'com.google.firebase:firebase-ads:9.8.0'
    compile 'com.firebaseui:firebase-ui:1.0.0'
    compile 'com.fasterxml.jackson.core:jackson-core:2.8.0'
    compile 'com.fasterxml.jackson.core:jackson-annotations:2.8.0'
    compile 'com.fasterxml.jackson.core:jackson-databind:2.8.0'
    compile 'com.amulyakhare:com.amulyakhare.textdrawable:1.0.1'
    compile 'org.apmem.tools:layouts:1.10@aar'
    compile 'com.github.bumptech.glide:glide:3.7.0'
    compile 'jp.wasabeef:glide-transformations:2.0.1'
    compile 'com.github.PhilJay:MPAndroidChart:v2.2.5'
    debugCompile 'com.squareup.leakcanary:leakcanary-android:1.5'
    releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5'
    testCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.5'
```

Where is the problem?

**Comment by Ashok-Varma** [comment]
@kirtan403 i am facing same issue. any fix you found ??

**Comment by Ashok-Varma** [comment]
got the issue my minSDK version is 15 => libraries minSdk is 16.

we need minSdk to be 16 or greater.

**Comment by kirtan403** [comment]
@Ashok-Varma My minSdk is 16. I still face the same issue. What I did is copy the same Activity declaration in my app's Manifest file (temp solution I know)

I added :
```
<activity
            android:name="com.facebook.FacebookActivity"
            android:configChanges="keyboard|keyboardHidden|screenLayout|screenSize|orientation"
            android:label="@string/app_name"
            android:theme="@style/FirebaseUI.Translucent"
            tools:replace="android:theme" />
```

to my manifest file, and resolved for now.

**Comment by Ashok-Varma** [comment]
@kirtan403 thankyou. once i removed facebook and changed minSdk it solved for me. i might loose 1.3% of users from changing 15 to 16 but it's ok

**Comment by kirtan403** [comment]
@Ashok-Varma Welcome 😄  I'm already on 16. But still didn't worked for me.

**Comment by SUPERCILEX** [comment]
@samtstern I believed this was fixed in #440 at this line: https://github.com/firebase/FirebaseUI-Android/commit/d783f197052c1528c18a4a64408bc2311b4349e9#diff-72b30eb5e208275c795ce7e9fb7275aeL77. (That commit is still in the dev branch so this issue can't be closed, but it can be marked as `fix-implemented`.)

**Comment by samtstern** [comment]
A fix for this issue has been released in version `1.1.0`
